src/common/components/modals/DoubleCheckModal.vue

```diff
@@ -44,7 +44,7 @@
                     <p-text-input v-model.trim="inputText"
                                   :invalid="invalid"
                                   block
-                                  @keyup.enter="confirm()"
+                                  @keyup.enter="handleConfirm(inputText)"
```

Comment:
question: Is it correct to enter `inputText` as the argument of `handleConfirm` function?